src/coverlet.core/Symbols/CecilSymbolHelper.cs

```diff
@@ -413,6 +413,7 @@ public static List<BranchPoint> GetBranchPoints(MethodDefinition methodDefinitio
             bool isAsyncStateMachineMoveNext = IsMoveNextInsideAsyncStateMachine(methodDefinition);
             bool isMoveNextInsideAsyncStateMachineProlog = isAsyncStateMachineMoveNext && IsMoveNextInsideAsyncStateMachineProlog(methodDefinition);
             bool skipFirstBranch = IsMoveNextInsideEnumerator(methodDefinition);
+            bool skipSecondBranch = false;
```

Comment:
Can you add a comment above `skipFirstBranch` like
```
// State machine for enumerator is handled with brfalse/beq or switch opcode and it depends on how many 'yield' we have in method body.
// For more than one 'yield' a switch is emitted so we can skip only first branch.
// In case of one 'yield' we need to skip first two branches to avoid to report phantom branch, first branch to jump to 'yield' value, second one to exit from enumeration
```

Reply:
> Some more tests, seems ok.

Yup, that's right. Will also add some more unit tests.